CHANGELOG.md

```diff
@@ -1,11 +1,12 @@
 CHANGELOG
 ---------
 
-## NEXT RELEASE
+## NEXT RELEASE
  - Improve DEVELOPPING.md with better testing command.
  - Fix an issue that make the Template Whisperer impossible to change in a entity.
  - Add possibility to chose which Suggestion are avaialble for selection, by field.
  - Add more tests to improve stability.
+ - Add new permission "administer the template whisperer field" for edit access of field(s).
```

Comment:
the new permission `administer the template whisperer field` is kind of a BC Break.
It would be nice to document the fact that since that release users need to have the permission to see the field.

Reply:
Yeah, I would do a hook_update but I don't know what to do, maybe use the old permission. List all users with `administer template whisperer` or `administer template whisperer suggestion entities` & add the new permission. What do you think about this process to avoid BC ?